src/test/java/org/graylog/integrations/notifications/types/SlackClientTest.java

```diff
@@ -49,14 +40,38 @@ public void tearDown() throws IOException {
     }
 
 
-
     @Test
     public void send_is_successful() throws Exception {
         SlackClient slackClient = new SlackClient(mockHttpClient);
         SlackMessage message = new SlackMessage("Henry Hühnchen(little chicken)");
         slackClient.send(message,"http://url.com");
```

Comment:
Currently this test only verifies that `slackClient.send()` didn't throw an exception.  You should add some checks to verify that it passed the expected values to OkHttp.
```suggestion
        slackClient.send(message,"http://url.com");
        
        ArgumentCaptor<Request> requestCaptor = ArgumentCaptor.forClass(Request.class);
        verify(mockHttpClient, times(1)).newCall(requestCaptor.capture());
        
        assertThat(requestCaptor.getValue(), notNullValue());
        Request sent = requestCaptor.getValue();
        assertThat(sent.url().toString(), is("http://url.com"));
        assertThat(sent.method(), is("POST"));
        assertThat(sent.body(), notNullValue());
        assertThat(sent.body().contentLength(), is(Long.valueOf(message.length())));
```